packages/pixel_pusher/pixel_pusher.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Control LED strips on Pixel Pusher hardware"
+description: """
+Light up LED strips by driving Pixel Pusher hardware modules over UDP/IP networks
+"""
+maintainer: "Michael Bacarella <michael.bacarella@gmail.com>"
+authors: ["Michael Bacarella <michael.bacarella@gmail.com>"]
+homepage: "https://github.com/mbacarella/pixel_pusher"
+bug-reports: "https://github.com/mbacarella/pixel_pusher/issues"
+dev-repo: "git+https://github.com/mbacarella/pixel_pusher.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "async_udp"   {>= "v0.12.0"}
+  "core"        {>= "v0.12.0"}
+  "bitstring"   {>= "v3.1.1"}
```

Comment:
```suggestion
  "bitstring"   {>= "3.1.1"}
```
The `v` prefix is only used for a handful of packages (generally speaking, only JaneStreet packages)